contents/handbook/engineering/ee-setup.md

```diff
@@ -4,6 +4,8 @@ sidebar: Handbook
 showTitle: true
 ---
 
+This page is deprecated. Please read [Developing Locally](/docs/contribute/developing-locally) instead.
```

Comment:
Why don't we delete this instead and just add a redirect?

Reply:
done!